### ISSUE TITLE
Release recent VEDA UI improvements (4.3.0)

### ISSUE THREAD

**Issue by j08lue** [NASA-IMPACT/veda-config#392]
The GHG Center Release 2 will publish feature improvements to the data catalog (floating filter sidebar) and the raster time series exploration (Exploration & Analysis).

These features should also be made available on the EarthData Dashboard, which is our testing and demo site, so we can use it to test further enhancements. 

However, there is some content preparation needed for the dataset and layer summary information that E&A surfaces.

## Depends on 

- [ ] https://github.com/NASA-IMPACT/veda-config/issues/391

## Acceptance criteria

- [ ] Released features from VEDA UI >=4.3.0

**Comment by aboydnw** [comment]
@j08lue is this layer info something we have? Or do we need to reach out to science teams for them to provide content?

**Comment by j08lue** [comment]
I think most info we can distil from dataset overview pages and layer descriptions. E.g. units are already part of the legend config. So it just takes someone to sieve through these texts and add the new config blocks.

I discussed this yesterday with @brianmfreitag - we do not need the info for all layers to release E&A. Let us see which we can put together quickly and which are most important (e.g. NO₂) and ticket the rest for follow-up.

**Comment by hanbyul-here** [comment]
Any reason that we need to use v4.4? Would it be ok to update to the latest of UI? (v5.1.1)?

**Comment by aboydnw** [comment]
Good call, I think that was just the latest version at the time the ticket was written. I updated it to be more generic, we should update to the latest version
